k8s/cmd/commands/flags/vineyardd_flags.go

```diff
@@ -36,6 +36,12 @@ var (
 
 	// VineyardSpillPVandPVC is PersistentVolume data and PersistentVolumeClaim data of vineyardd's spill mechnism
 	VineyardSpillPVandPVC string
+
+	// VineyardFile is the path of vineyardd file
+	VineyarddFile string
+
+	// WaitVineyardd is the flag to indicate whether to wait for the vineyardd to be ready
+	WaitVineyardd bool
```

Comment:
Have a global/reuseuable "wait" parameter would be better. I think.

Reply:
Make sense.